umm-spec-lib/src/cmr/umm_spec/iso19115_2_util.clj

```diff
@@ -13,6 +13,14 @@
 (def short-name-xpath
   "gmi:identifier/gmd:MD_Identifier/gmd:code/gco:CharacterString")
 
+;; relative to gmi:childOperation
+(def campaign-xpath
+  "gmi:MI_Operation/gmi:identifier/gmd:MD_Identifier/gmd:code/gco:CharacterString") 
+
+(def start-end-date-xpath
```

Comment:
I would recommend you remove it. The xpath isn't long enough to make the case for being more easily read and it is only used once.  I think it would be better to just not use a def for this.